multiqc/modules/multivcfanalyzer/multivcfanalyzer.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from MultiVCFAnalyzer """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import json
+
+from multiqc.plots import table
+from multiqc.plots import bargraph
+from collections import OrderedDict
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """ MultiVCFAnalyzer module """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='MultiVCFAnalyzer', anchor='multivcfanalyzer',
+        href="https://github.com/alexherbig/MultiVCFAnalyzer",
+        info="""combines multiple VCF files in a coherent way, can produce summary statistics and downstream analysis formats for phylogeny reconstruction.""")
+
+        # Find and load any MultiVCFAnalyzer reports
+        self.mvcf_data = dict()
+
+        # Find and load JSON file
+        for f in self.find_log_files('multivcfanalyzer', filehandles=True):
+            self.parse_data(f)
+
+        # Filter samples
+        self.mvcf_data = self.ignore_samples(self.mvcf_data)
+
+        # Return if no samples found
+        if len(self.mvcf_data) == 0:
+            raise UserWarning
+
+        # Add in extra columns to data file
+        self.compute_perc_hets()
+
+        # Save data output file
+        self.write_data_file(self.mvcf_data, 'multiqc_multivcfanalyzer')
+
+        # Add to General Statistics
+        self.addSummaryMetrics()
+
+        # Add MultiVCFAnalyzer Table Section
+        self.add_section (
+        name = 'MultiVCFAnalyzer analysis results',
+        anchor = 'mvcf_table',
+        helptext = "This is the output from MultiVCFAnalyzer. Please refer to the manual here - https://github.com/alexherbig/MultiVCFAnalyzer for more information on how to interpret data.",
```

Comment:
This help text doesn't really, well, _help_ much 😉 

Either describe the columns in the table - what they mean, what could be expected, when to worry. Or alternatively please remove this line as it's essentially a duplicate of the text above it.